### ISSUE TITLE
[Accessibility icon][desktop] Accessibility icon is not visible for brave browser with native adblock

### ISSUE THREAD

**Issue by galeza** [FightPandemics/FightPandemics#2118]
env - https://fightpandemics.com/
browser - brave
with native adblock

When native adblock is on - accessibility icon is not visible. Please see screenshots:
![Screen Shot 2021-01-04 at 15 21 41](https://user-images.githubusercontent.com/21987083/103544582-9cf57d00-4ea0-11eb-82d2-4fd8d74e7f3c.png)
![Screen Shot 2021-01-04 at 15 21 48](https://user-images.githubusercontent.com/21987083/103544595-9ff06d80-4ea0-11eb-85b6-328e5b5eb87a.png)

For firefox and adblock addon I didn't see this issue. Icon is visible when Adblock is on:
![Screen Shot 2021-01-04 at 15 13 28](https://user-images.githubusercontent.com/21987083/103544640-b4cd0100-4ea0-11eb-919c-7575dced1228.png)
